Dockerfile

```diff
@@ -1,34 +1,43 @@
-FROM ubuntu:latest
+FROM ubuntu:20.04
```

Comment:
in other repos we are still using 18.04, but physics is likely on 20, so good choice. This could cause fortran validation issues, unless that gets installed with the same OS, so be on the lookout. Maybe your fortran build is using 20 though.

Reply:
This is what Florian recommended to fix the problem with xarray, it is now the version in master of physics_standalone